modules/cli/src/main/scala/scala/cli/commands/shared/SharedVersionOptions.scala

```diff
@@ -0,0 +1,26 @@
+package scala.cli.commands.shared
+
+import caseapp.*
+
+import scala.cli.commands.tags
+
+// format: off
+final case class SharedVersionOptions(
+  @Group(HelpGroup.ProjectVersion.toString)
+  @HelpMessage("Method used to compute the project version")
+  @ValueDescription("git|git:tag|command:...")
+  @Tag(tags.restricted)
+  @Tag(tags.inShortHelp)
+    computeVersion: Option[String] = None,
+  @Group(HelpGroup.ProjectVersion.toString)
+  @HelpMessage("Set the project version")
+  @Tag(tags.restricted)
+  @Tag(tags.inShortHelp)
+    projectVersion: Option[String] = None
```

Comment:
Add an alias `version` to keep backward compatibility.

Reply:
Can't do that, after refactoring this option into SharedOptions the name had to be changed to avoid clashing with `scala-cli --version` as the Default command has a `version` flag.

Reply:
Right, I hadn't checked that. So, it seems fine to me.